usim/_primitives/activity.py

```diff
@@ -51,16 +51,37 @@ def __transcript__(self) -> ActivityCancelled:
 
 
 class ActivityExit(BaseException):
-    ...
+    """A :py:class:`~.Activity` forcefully exited"""
 
 
-class Activity(Condition, Generic[RT]):
+class Activity(Awaitable[RT]):
     """
-    Active coroutine that allows others to listen for its completion
+    Concurrently running activity that allows multiple activities to await its completion
 
-    :note: Simulation code should never instantiate this class directly.
+    A :py:class:`Activity` represents an activity that is concurrently run in a :py:class:`~.Scope`.
+    This allows to store or pass an an :py:class:`Activity`, in order to check its progress.
+    Other activities can ``await`` a :py:class:`Activity`,
+    which returns any results or exceptions on completion, similar to a regular activity.
+
+    .. code:: python3
+
+        await my_activity()  # await a bare activity
+
+        async with Scope() as scope:
+            activity = scope.do(my_activity())
+            await activity   # await a rich activity
+
+    In contrast to a regular activity, it is possible to
+
+    * :py:meth:`~.Activity.cancel` an :py:class:`Activity` before completion,
+    * ``await`` the result of an :py:class:`Activity` multiple times,
+      and
+    * ``await`` that an is an :py:class:`Activity` is :py:attr:`~.Activity.done`.
```

Comment:
There seems to be a word missing between "an ... is an". So this part of the list currently is not clear to me.